apps/prairielearn/elements/pl-excalidraw/pl-excalidraw.py

```diff
@@ -0,0 +1,96 @@
+import base64
```

Comment:
Make sure to add this file to the pyright section in `pyproject.toml` to enable type checking.